src/plugins/foxPage/FoxCommon.ts

```diff
@@ -0,0 +1,61 @@
+import { TradeOpportunitiesBucket } from './components/TradeOpportunities'
+
+export const FOX_ASSET_ID = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
+export const FOXY_ASSET_ID = 'eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3'
+export const TrimmedDescriptionLength = 191
+
+export enum OpportunityTypes {
+  LiquidityPool = 'liquidityPools',
+  Farming = 'farming',
+  BorrowingAndLending = 'borrowingAndLending',
+}
+
+export type ExternalOpportunity = {
+  title: string
+  apy?: string | null
+  link: string
+  icons: string[]
+}
+
+export type OpportunitiesBucket = {
+  type: OpportunityTypes
+  title: string
+  opportunities: ExternalOpportunity[]
+}
+
+export const foxTradeOpportunitiesBuckets: TradeOpportunitiesBucket[] = [
+  {
+    title: 'plugins.foxPage.dex',
+    opportunities: [
+      {
+        link: 'https://app.uniswap.org/#/swap?inputCurrency=ETH&outputCurrency=0xc770eefad204b5180df6a14ee197d99d808ee52d&chain=mainnet',
+        icon: 'uniswap.png',
```

Comment:
Note: I've decided to keep these while taking it over, passing the icon path down like this is serializable so it makes React happy